exercises/01-composite-ui/before/Divergent.Customers.API/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Owin.Hosting;
+
+namespace Divergent.Customers.API
+{
+    class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var tcs = new TaskCompletionSource<object>();
+            Console.CancelKeyPress += (sender, e) => { tcs.SetResult(null); };
+
+            using (WebApp.Start<Startup>("http://localhost:20186"))
+            {
+                await Console.Out.WriteLineAsync("Divergent.Customers.API web server is running.");
```

Comment:
Minor point, but I'm not sure it's worth repeating the namespace here, since the console window will already contain the name of the app. Or if we do, it would be better read it from metadata to save having to repeat it here. It could also be good to put the listening URL into a variable and print it to the console. E.g. `Divergent.Customers.API listening at http://localhost:20186`